src/workloads/execution/FilterWithLogicalExpressions.yml

```diff
@@ -160,3 +336,40 @@ Actors:
                 ]
               }
             }
+
+- Name: MatchExpressionsMixedPredicate
+  Type: CrudActor
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [9]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {
+              $or: [
+                {$and: [{a: 10}, {b: 11}]},
+                {$and: [{c: 12}, {d: 13}]},
+                {a: 14},
+                {b: 15},
+                {c: 16},
+                {d: 17},
+              ]
+            }
+
+AutoRun:
+- When:
+    mongodb_setup:
+      $eq:
+      - replica-all-feature-flags
```

Comment:
Ah, this isn't quite right. My recommendation would be to run it in four configurations, though I'm also curious what people from the perf team think:

- vanilla standalone
- standalone, all feature flags
- standalone, SBE
- standalone, classic

The first two will give us value going forward. The SBE and classic variants are necessary because we are currently focusing on this comparison for the SBE perf project.

Reply:
lgtm

Reply:
Sounds good, done.

Reply:
I left a new comment about this, but I'll resolve this one to keep the comment threads cleaner.